src/components/views/elements/MemberEventListSummary.js

```diff
@@ -277,6 +277,11 @@ module.exports = React.createClass({
                     ? _t("%(severalUsers)schanged their avatar %(count)s times", { severalUsers: "", count: repeats })
                     : _t("%(oneUser)schanged their avatar %(count)s times", { oneUser: "", count: repeats });
                 break;
+            case "no_change":
+                res = (userCount > 1)
```

Comment:
I think we would also want to test the setting value here as well, so that we don't show this text unless hidden events are enabled...?

Reply:
https://github.com/matrix-org/matrix-react-sdk/blob/8394e162ad7d1b8b5b253b81401bce2e70518a47/src/components/structures/MessagePanel.js#L416 handles that before MELS even gets the event

Reply:
Ah right, thanks for pointing this out!